graph/graph_windows.go

```diff
@@ -114,7 +119,29 @@ func (graph *Graph) storeImage(img *Image, layerData archive.ArchiveReader, root
 
 		// Store the layer. If layerData is not nil, unpack it into the new layer
 		if layerData != nil {
-			if img.Size, err = graph.driver.ApplyDiff(img.ID, img.Parent, layerData); err != nil {
+			// this is saving the tar-split metadata
```

Comment:
@vbatts can we extract what's inside the `if layerData` into a function to not duplicate the code?